_posts/2020-07-21-catalogs.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title: "Sharing is caring - Catalogs in Flink SQL"
+date: 2020-07-21T08:00:00.000Z
+authors:
+- dawid:
+  name: "Dawid Wysakowicz"
+  twitter: "dwysakowicz"
+---
+
+It's not a surprise that, in an era of digitalization, data is the most valuable asset in many companies: it's always the base for — and product of — any analysis or business logic. With an ever-growing number of people working with data, it's a common practice for companies to build self-service platforms with the goal of democratizing their access across different teams and — especially — to enable users from any background to be independent in their data needs. In such environments, metadata management becomes a crucial aspect. Without it, users often work blindly, spending too much time searching for datasets and their location, figuring out data formats and similar cumbersome tasks.
```

Comment:
Maybe it's just my personal taste, but "era of digitalization" and "data is the most valuable asset" sounds like some marketing wording, ringing some alarm bells. If this text would not be published on the Flink blog, I would stop reading here.

I guess the intention here is to motivate the relevance of the catalog integrations in Flink. Maybe a more technical motivation is better suited for the Flink blog? I find it quite exciting that you can use Flink as a universal query engine for data stored in Postgres or tables defined in the Hive Metastore.

Reply:
I will try to rephrase it to be less "marketing".

Reply:
I though about it again this morning. I removed the first sentence with the "marketing" wording.

I did not further change the introduction. I understand your point of "using Flink as a universal query engine..." My idea for this blog was slightly different though, with more stress on the relevance of catalogs as such. That's the core idea I build around. Therefore the introduction is not Flink specific and less technical. Only once I explain the relevance of catalogs I go into the Flink catalogs support. This is to showcase that Flink does well on that important topic of catalogs that I explain in the beginning.

Reply:
See my other comment. I realized that my expectations about the blog post were not properly managed :)